Xamarin.Forms.Platform.Android/Renderers/ShellFlyoutRecyclerAdapter.cs

```diff
@@ -43,7 +42,22 @@ public ShellFlyoutRecyclerAdapter(IShellContext shellContext, Action<Element> se
 
 		public override int GetItemViewType(int position)
 		{
-			var item = _listItems[position];
+			return _listItems[position].Index;
```

Comment:
This is probably fine, but I'm curious why this instead of just returning the position passed in which will also always be unique?